lm75bd/lm75bd.c

```diff
@@ -27,7 +27,25 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
-  
+
+  uint8_t pointerReg[1] = {0};
+  i2cSendTo(LM75BD_OBC_I2C_ADDR, pointerReg, 1);
```

Comment:
Also, make sure to check the return value of the functions. i2cSendTo returns an error code. You can use RETURN_IF_ERROR_CODE macro found in sys/logging/logging.h.